cumulusci/cumulusci.yml

```diff
@@ -110,6 +110,10 @@ tasks:
         description: Retrieves a list of the currently installed managed package namespaces and their versions
         class_path: cumulusci.tasks.salesforce.GetInstalledPackages
         group: Salesforce Metadata
+    github_parent_pr_notes:
+        description: Merges the description of a child PRs to the respective Parent's PR (if one exists).
+        class_path: cumulusci.tasks.release_notes.task.ParentPullRequestNotes
+        group: GitHub
```

Comment:
We need to talk through how this will fit into our MetaCI Plans. It probably makes sense to add it to the ci_feature flow which is triggered on every commit to a feature branch. That has some implications: The task can assume it's being called from a feature branch, but the flow won't be able to work out the parent & child branch names to pass to the task. The task will need to look at self.project_config.repo_branch to know what the current branch is, and determine whether it's a parent or child (based on whether it has a double-underscore or not) and behave accordingly.

Reply:
Okay, that sounds good and I think I was just a bit confused.

You should be able to add a step like this to the ci_feature flow yaml to pass in the branch name:

    task: github_parent_pr_notes
    options:
        branch_name: $project_config.repo_branch